### DISCUSSION TITLE
Respecto al informe

### DISCUSSION THREAD

**Discussion by jalonsoluc** [IIC2133-PUC/2024-1#29] [Tarea 0] [answered]
Hola! Cómo están?

¿Se puede **eliminar la sección de referencias en el informe** si ésta está vacía? ¿O se debe incluir, y especificar que no se hicieron referencias? Si se incluye, ¿se considera en las dos planas máximas de extensión?

Gracias!

**Comment by sachondo7** [comment, marked as answer]
Las referencias no valen en la extensión de las 2 páginas. Sugiero dejarlo en blanco para no modificar el template
